build/Versions.props

```diff
@@ -74,6 +74,7 @@
     
     <!-- Libs -->
     <NewtonsoftJsonVersion>9.0.1</NewtonsoftJsonVersion>
+    <SystemReflectionMetadataVersion>1.4.2</SystemReflectionMetadataVersion>
```

Comment:
>1.4.2< [](start = 37, length = 6)

Why is this 1.4.2 and not 1.5.0?

Reply:
This is the same version Roslyn is on. 1.5.0 takes a dependency on a newer version of System.Collections.Immutable than we are using, too.